src/Adapter/LegacyLogger.php

```diff
@@ -137,6 +137,7 @@ public function debug($message, array $context = [])
      */
     public function log($level, $message, array $context = [])
     {
+        $pslevel = 0;
```

Comment:
Prefer adding this in the `default` switch statement

Reply:
:heavy_check_mark: